### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Access Keys_Keyboard: Focus is not visible on controls(Create Access Key, Refresh, Delete) of page. Issue is observed through out the application. Focus is not visible on any of the controls.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#56]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites; #Keyboard; #WCAG2.4.7; #C+E-H; #Chrome; #Edge

**Environment Details:**
URL: https://localhost:55539/security/tokens
Browser: Edge(42.17134.1.0), Chrome(68.0.3440.106)
Keyboard.

**Repro Steps:**
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till "Skip This" button and Press Enter.
4.Tab till "Get Access Token" link and Press Enter.
5. Tab to all controls on page and verify focus is visible on all controls of page.

**Actual Result:**
Focus is not visible on few controls(Create Access Key, Refresh, Delete) of page.

**NOTE: Same issue is observed in below pages:**

- Access Key Purpose
- Access Token
- Mime Maps
- Logging
- Central Certificate Store
- Direct Browsing

**Expected Result:**
Keyboard focus should be clearly visible when focus is on any control of page.

**User Impact:**
Keyboard only users find difficulty in using the application if keyboard focus is not visible on any control

**Attachment:**
![bug_56_access keys_focus visible issue](https://user-images.githubusercontent.com/38523250/44526713-7d5fbe80-a702-11e8-9f62-99fa1bd51250.png)

**Comment by Amlan454** [comment]
same issue is observed in application pool edit also
![bug_56](https://user-images.githubusercontent.com/42638683/44908670-f336dc00-ad39-11e8-9653-8c6aa8cb1b40.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by fsteffi** [comment]
#A11ySev1;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
